docs/using.adoc

```diff
@@ -208,41 +209,24 @@ OpenShift.
 === Using `docker`
 
 ADB provides a full container environment and runs both `docker` and
-`kubernetes`. All standard commands work, for example:
-
-....
-ADB pull centos
- docker run -t -i centos /bin/bash
-....
+`kubernetes`, with most standard commands. 
```

Comment:
@LalatenduMohanty ping again re: Budhram's request, can you help with context here please?